README.md

```diff
@@ -1,24 +1,15 @@
-    :width: 60%
-
-.. image:: https://img.shields.io/pypi/wheel/nucypher.svg
-    :target: https://pypi.org/project/nucypher/
+![](/docs/source/.static/img/nucypher.png)
 
-.. image:: https://img.shields.io/pypi/pyversions/nucypher.svg
-    :target: https://pypi.org/project/nucypher/
-
-.. image:: https://coveralls.io/repos/github/nucypher/nucypher/badge.svg?branch=master
-    :target: https://coveralls.io/github/nucypher/nucypher?branch=master
+*A proxy re-encryption network to empower privacy in decentralized systems*
 
-.. image:: https://circleci.com/gh/nucypher/nucypher/tree/master.svg?style=svg
-    :target: https://circleci.com/gh/nucypher/nucypher/tree/master
+----
 
+![](https://img.shields.io/pypi/wheel/nucypher.svg)
+![](https://img.shields.io/pypi/pyversions/nucypher.svg)
+![](https://coveralls.io/repos/github/nucypher/nucypher/badge.svg?branch=master)
+![](https://circleci.com/gh/nucypher/nucypher/tree/master.svg?style=svg)
+![](/docs/source/.static/img/GPLv3.png)
```

Comment:
How about ![](/docs/source/.static/img/GPLv3.png) -> ![](https://img.shields.io/pypi/l/nucypher.svg)

Reply:
Why does it say missing? I just use the GPLv3 logo. It's not like we need to show the status of our license lol

Reply:
It says missing because the package it not yet on PyPI - No - but I prefer the uniformity pf the badges

Reply:
Plus - that logo looks like a 90's power ranger

Reply:
Let's do it incrementally, once we get the package on PyPI then let's switch to that badge.

Reply:
I don't want it to say "license: missing" when it really shouldn't.

Reply:
Well - we already do this in several places in the docs (the other badges for example), and I anticipate the period of time before publication to be small - I think a follow-up PR on this basis is overkill.